spec/runner_spec.rb

```diff
@@ -159,10 +159,11 @@ def all_request_bodies
   end
 
   describe '.runners' do
-    context 'when filtering' do
+    context 'when filtering with cli option' do
       it 'returns everything by default' do
         Jacaranda.parse([])
-        expect(Jacaranda.runners.size).to be >= mock_runner_count
+        runners = Jacaranda.runners & mock_runners
```

Comment:
I do find these tests a bit confusing. It seems like `.parse` is effectively being tested throughout here as well?

It's confusing that we're testing `.runners` but then using the [`&` method](http://ruby-doc.org/core-2.4.1/Array.html#method-i-26) to pick out the common elements in these two arrays, manipulating the result of the execution before we verify them.